### PR TITLE
[SC-8564] RabbitMQ logs integration

### PR DESCRIPTION
**Pull request by radu-gheorghe**

### CODE REVIEW

File: docs/integration/rabbitmq-logs.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+/>
+
+## Setting up Logagent
+
+With [Node.js installed](https://nodejs.org/en/download/package-manager/), you'd first need to install Logagent:
+```bash
+sudo npm i -g @sematext/logagent
+```
+
+Then, write a config file that tails your RabbitMQ logs and sends them to your RabbitMQ Logs App. Parsing happens out of the box, since we match the `rabbit` source:
```
Comment (otisg) [inline docs/integration/rabbitmq-logs.md:22]: Is it "rabbit" or "rabbitmq"?

File: docs/integration/rabbitmq-logs.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      YOUR_LOGS_TOKEN_GOES_HERE:
+        - .*rabbit.*\.log
+```
+
+Finally, use `logagent-setup` to copy the config to `/etc/sematext/logagent.conf`, then set up the init script and start Logagent:
+```bash
+sudo logagent-setup -c /path/to/logagent.conf
+```
+
+If you already have Logagent installed, you an simply append to the `files` input the RabbitMQ log file, then the two lines (token+pattern) in the `elasticsearch` output. In the end, restart Logagent.
```
Comment (otisg) [inline docs/integration/rabbitmq-logs.md:50]: an => can

File: docs/integration/rabbitmq-logs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  class="content-modal-image"
+  alt="RabbitMQ Authentication Report"
+  src="../../images/agents/rabbitmq_auth.png"
+  title="RabbitMQ Authentication Report"
+/>
+
+Other built-in reports include:
+
+- **Connections**: Logs specifically about accepted and closed connections. You can see here when you have spikes in connection creation. You can also identify noisy sources, users, as well as popular destinations.
+- **Start & Stop**: Logs about RabbitMQ startup and shutdown. Besides unexpected restarts, you can find info about which write-ahead logs (WALs) were recovered
```
Comment (otisg) [inline docs/integration/rabbitmq-logs.md:66]: . at the end

File: docs/integration/rabbitmq-logs.md (hunk 1 above)
Comment (radu-gheorghe) [reply docs/integration/rabbitmq-logs.md:22]: it is `rabbit` because logs can look like `rabbit@ubuntu2004.log`

**Pull request merged**